cabot/metricsapp/api/base.py

```diff
@@ -150,18 +122,19 @@ def run_metrics_check(check):
         message = series.get('error_message')
         logger.exception('Error fetching metrics: {}: {}'.format(series.get('error_code'), message))
         error = 'Error fetching metric from source: {}'.format(message)
-        return StatusCheckResult(check=check, succeeded=False, error=error)
+        return StatusCheckResult(check=check, succeeded=False, error=error), ['fetch_error']
 
     # If the series is empty, apply the empty-series handler
     if series['data'] == []:
         if check.on_empty_series == defs.ON_EMPTY_SERIES_PASS:
-            return StatusCheckResult(check=check, succeeded=True, error='SUCCESS: no data')
+            return StatusCheckResult(check=check, succeeded=True, error='SUCCESS: no data'), []
         if check.on_empty_series == defs.ON_EMPTY_SERIES_WARN:
             check.importance = Service.WARNING_STATUS
-            return StatusCheckResult(check=check, succeeded=False, error='WARNING: no data')
+            return StatusCheckResult(check=check, succeeded=False, error='WARNING: no data'), ['warning:no data']
         if check.on_empty_series == defs.ON_EMPTY_SERIES_FAIL:
             check.importance = check.high_alert_importance
-            return StatusCheckResult(check=check, succeeded=False, error='{}: no data'.format(check.importance))
+            tags = ['{}:no data'.format(check.importance.lower())]
```

Comment:
nit: Could we just have a generic 'no data failure' for these? I can't really think of a case where we'd need multiple tags at different levels for this since you're only allowed to pick one option, so doesn't seem entirely necessary to distinguish?

Reply:
agreed!